src/component/timeBlock/molecules/TimeList.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import { Time } from '@/component/timeBlock/atom';
+import { styled } from 'styled-components';
+
+interface TimeListProps {
+  start?: string;
+  end?: string;
+}
+
+export const TimeList = ({ start = '9:00', end = '17:00' }: TimeListProps) => {
+  const [timeList, setTimeList] = useState(['']);
```

Comment:
요건 차라리 아래 코드면 어떨까 의견 내봅니다! array of string 의 기본값은 배열 안에 아무것도 없는거니까요!
```suggestion
  const [timeList, setTimeList] = useState<string[]>([]);
```